### PR TITLE
Update README to reflect new Node version guidance

### PR DESCRIPTION
**Pull request by turnerhayes**

### CODE REVIEW

**Comment by turnerhayes** [comment]
I probably should have done this in my fork; I did this via the GitHub UI.

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ To follow these instructions, it'd help if you:
 Instructions:
 1. If you're new to github look at [How to: fork a repo](https://help.github.com/articles/fork-a-repo/). Otherwise, fork and clone the repo, creating a link (remote branch) to the Code for Boston repo.
 2. In your termainal/command prompt, do `git checkout dev`. *Do not touch the `master` branch.*
-3. If you have [nodejs](https://nodejs.org/en/), version 5.0+ is desired. If you don't, [install it](https://nodejs.org/en/download/). If you don't know, do `node --version` in your terminal/command prompt. If you don't get a number, you don't have it.
+3. If you have [nodejs](https://nodejs.org/en/), version v10.13.0 is desired. If you don't, [install it](https://nodejs.org/en/download/). If you don't know, do `node --version` in your terminal/command prompt. If you don't get a number, you don't have it. You can also install [NVM](https://github.com/creationix/nvm) and run `nvm use` in your terminal to use the correct version (you may have to `nvm install` if you don't currently have the correct version installed).
```
Comment (knod) [inline README.md]: 'version v10.13.0' is redundant, but if you don't want to change it, I'll do it later.

**Review by knod — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (turnerhayes) [reply README.md]: True! Fixed in branch.

**Comment by knod** [comment]
Yeah, I try to avoid the UI when possible.

**Pull request merged**